int8 axis
float64 value
---
bool success
string message
